Do not foolishly attempt to download jars from the internets.

diff -up xmlbeans-2.4.0/build.xml.nodownload xmlbeans-2.4.0/build.xml
--- xmlbeans-2.4.0/build.xml.nodownload	2008-06-10 15:42:37.000000000 +0200
+++ xmlbeans-2.4.0/build.xml	2009-11-03 15:53:11.000000000 +0100
@@ -112,7 +112,7 @@
             http://easynews.dl.sourceforge.net/sourceforge/saxon/saxonb8-7-3j.zip
             http://easynews.dl.sourceforge.net/sourceforge/saxon/saxonb8-8j.zip
         -->
-        <get dest="external/lib/saxonb9-0-0-4j.zip"
+        <!-- get dest="external/lib/saxonb9-0-0-4j.zip"
             src="http://prdownloads.sourceforge.net/saxon/saxonb9-0-0-4j.zip"
             verbose="true" usetimestamp="true" ignoreerrors="true"/>
         <unzip src="external/lib/saxonb9-0-0-4j.zip" dest="external/lib/">
@@ -120,7 +120,7 @@
                 <include name="saxon9.jar"/>
                 <include name="saxon9-dom.jar"/>
             </patternset>
-        </unzip>
+        </unzip-->
 
         <copy file="external/lib/saxon9.jar" tofile="build/lib/saxon9.jar"/>
         <!-- saxon9-dom.jar is used to register DOMObjectModel -->
@@ -147,14 +147,14 @@
         </condition>
     </target>
 
-    <target name="jsr173_1.0.jars"
+    <target name="jsr173_1.0.jars"><!--
             depends="jsr173-bundle.jar, jsr173_1.0.jars.extracted"
             unless="jsr173_1.0.jars.exist">
         <unjar src="external/lib/jsr173_1.0_api_bundle.jar" dest="external/lib" >
             <patternset>
                 <include name="jsr173_1.0_api.jar"/>
             </patternset>
-        </unjar>
+        </unjar!-->
 
         <!-- copy jsr173 API jar and license files to build/lib -->
         <copy file="external/lib/jsr173_1.0_api.jar" tofile="build/lib/jsr173_1.0_api.jar"/>
